2023-10-12/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/vmorsell/joel/2023-10-12/names"
```

Comment:
Importing the file names

Reply:
Almost, it's importing the _package_ names. Go actually doesn't care about file names. Try change the names of the file and the directory and see what happens
https://go.dev/doc/code#Organization